Sources and OutletItems (#408)
MewsSystems/gitbook-connector-api: operations/sources.md

| `ClientToken` | string | required | Token identifying the client application. |
| `AccessToken` | string | required | Access token of the client application. |
| `Client` | string | required | Name and version of the client application. |

> | `SourceIds` | array of string | optional, max 1000 items | Unique identifiers of [Source](sources.md#source). |

Reviewer: ```suggestion
| `SourceIds` | array of string | optional, max 1000 items | Unique identifiers of [Sources](sources.md#source). |

```

> | `UpdatedUtc` | [Time interval](#time-interval) | optional, max length 3 months | Interval in which the [Source](sources.md#source) was updated. |

Reviewer: It's a minor point, but it's nice if the order of the properties in the examples are the same as the order of the properties in the properties table. Here they are the other way around.

Author: It's a good point! 🚀

        {
            "Id": "bbe29c21-401a-4746-b97d-af1f00e1bb8b",
            "Name": "Booking.com",
            "Type": "OnlineTravelAgency",

>             "UpdatedUtc": "2023-01-05T12:00:00Z",

Reviewer 2: ```suggestion
            "UpdatedUtc": "2023-01-05T12:00:00Z"

```

        },
        {
            "Id": "22e42a59-b321-43f8-a5d1-af1f00e1bb8b",
            "Name": "Hostelworld",
            "Type": "OnlineTravelAgency",

>             "UpdatedUtc": "2023-01-06T12:00:00Z",

Reviewer 2: ```suggestion
            "UpdatedUtc": "2023-01-06T12:00:00Z"

```

        },
        {
            "Id": "b5a55d41-bbc5-48d0-a01b-af1f00e1bb8b",
            "Name": "SynXis",
            "Type": "OnlineTravelAgency",

>             "UpdatedUtc": "2023-01-06T12:00:00Z",

Reviewer 2: ```suggestion
            "UpdatedUtc": "2023-01-06T12:00:00Z"

```

| `Id` | string | required | Unique identifier of the source. |
| `Name` | string | required | Name of the source. |
| `Type` | string [Source type](#source-type) | required | Type of the source. |

> | `UpdatedUtc` | string | required | Last update date and time of the source in UTC timezone in ISO 8601 format. |

Reviewer: ```suggestion
| `UpdatedUtc` | string | required | Date and time when the source was last updated, expressed in UTC timezone in ISO 8601 format. |

```
